tests/functional/api/bulk/action_test.py

```diff
@@ -7,6 +7,8 @@
 
 from h.models import Group, GroupMembership, User
 
+pytestmark = pytest.mark.usefixtures("init_elasticsearch")
```

Comment:
Now that the `init_elasticsearch` fixture is no longer `autouse` any tests that need Elasticsearch need to have this.